x/superfluid/keeper/migrate_test.go

```diff
@@ -53,15 +53,28 @@ func (suite *KeeperTestSuite) TestRouteLockedBalancerToConcentratedMigration() {
 		"lock that is superfluid delegated, not unlocking (partial shares)": {
 			// migrateSuperfluidBondedBalancerToConcentrated
 			superfluidDelegated:      true,
-			percentOfSharesToMigrate: sdk.MustNewDecFromStr("0.5"),
+			percentOfSharesToMigrate: sdk.MustNewDecFromStr("0.4"),
```

Comment:
Out of curiosity, what motivated this change?

Reply:
Migrating exactly half made a nice number, and I wanted to ensure it wasn't just happenstance that the expected numbers aligned